build.sh

```diff
@@ -221,10 +223,11 @@ BUILD_DIR="./${BUILD_TYPE}-build"
 
 rm -rf "${BUILD_DIR}"
 mkdir "${BUILD_DIR}"
+[ "${BUILD_WITH_CLANG}" != "" ] && CMAKE_EXTRA_FLAGS="-DCMAKE_CXX_COMPILER=$LLVM_DIR/bin/clang++ -DLLVM_DIR=$LLVM_DIR"
 cmake -D CMAKE_BUILD_TYPE:STRING="${BUILD_TYPE}" \
```

Comment:
Can we add “ DCMAKE_CXX_COMPILER” here similar to this line? The basic principle is to make sure all macros are not ad hoc and should be as consistent as possible and reduce arbitrary insertions and try to keep it as simple as possible

Reply:
I might not quite follow, are u talking about the way https://github.com/SVF-tools/SVF/pull/1093/commits/f723d9de8925ee7cc4e9de57b4d720f4cfe535bd?